map/src/manager/track/TracksManager.js

```diff
@@ -1130,8 +1130,8 @@ export const getGpxTime = (f, reverse = false, creationTime = false) => {
         raw.push(f?.file?.clienttimems); // uploaded for favorites group(cloud timestamp?)
         raw.push(f?.ext?.time); // uploaded for favorites group(cloud timestamp?)
     } else {
-        if (f?.updatetimemsbywpts) {
-            raw.push(f?.updatetimemsbywpts);
+        if (isFavGroups) {
+            raw.push(f?.clienttimems);
```

Comment:
If it is possible that clienttimems is not defined (for example, old non-re-uploaded files maybe), please add condition like this: 

`raw.push(f?.clienttimems ?? f?.updatetimems);`